gpflow/quadrature.py

```diff
@@ -80,15 +80,18 @@ def mvnquad(func, means, covs, H: int, Din: int=None, Dout=None):
     return tf.reduce_sum(fX * wr, 0)
 
 
-def ndiagquad(funcs, H: int, Fmu, Fvar, **Ys):
+def ndiagquad(funcs, H: int, Fmu, Fvar, logspace: bool=False, **Ys):
```

Comment:
Could you fill all type annotations? https://docs.python.org/3/library/typing.html

Reply:
no, because I don't know how to describe the types - it could be tf.Tensor, tf.Variable, ... - maybe even a numpy array - and it doesn't make sense to have a complicated Union[] all over the place: if you add an appropriate type in gpflow I'm happy to add it here.